pallet-chainlink-feed/src/lib.rs

```diff
@@ -571,7 +589,51 @@ pub mod pallet {
 		// --- feed operations ---
 
 		/// Create a new oracle feed with the given config values.
-		/// Limited to feed creator accounts.
+		///
+		/// This will issue the next available feed ID (`FeedCounter`) and
+		/// assign it to the new feed.
+		///
+		/// Limited to feed creator accounts that where granted permission by
+		/// the pallet admin to create feeds via `set_feed_creator`.
+		///
+		/// Parameters:
+		/// - `payment`: The amount of funds an oracle should received per valid
+		///   round submission.
+		/// - `timeout`: The timeout in blocks, after which a started round is
+		///   timed out. If a round started in block `n` and the feed's
+		///   `timeout` is `10`, then the round is timed out if `n + 10 <
+		///   current_block`.
+		/// - `submission_value_bounds`: The value bounds of oracle submissions
+		///   that are considered valid.
+		/// - `min_submissions`: Determines the lower end of the feed's
+		///   `submission_count_bounds` where the maximum number of submissions
+		///   a round records is the number of configured oracles and the lower
+		///   end is the minimum amount of submissions a round needs to update
+		///   an answer via `AnswerUpdated` event.
+		/// - `decimals`: Metadata that represents the number of decimals with
+		///   which the feed is configured.
+		/// - `description`: A user friendly name or ticker of this feed.
+		///   Limited in length by `StringLimit`..
+		/// - `restart_delay`: The delay in blocks that needs to pass before a
```

Comment:
Maybe this should of type `u32`?

Reply:
yes, same thing as with `pruning_window`, this is also a range